internal/run/run.go

```diff
@@ -33,16 +34,19 @@ func (c cmdWithStderr) Output() ([]byte, error) {
 	if isVerbose, _ := utils.IsDebugEnabled(); isVerbose {
 		_ = printArgs(os.Stderr, c.Cmd.Args)
 	}
-	if c.Cmd.Stderr != nil {
-		return c.Cmd.Output()
-	}
-	errStream := &bytes.Buffer{}
-	c.Cmd.Stderr = errStream
```

Comment:
`Cmd.Output()` collects the command stderr output so utilize that mechanism rather than overriding `c.Cmd.Stderr`.